.travis.yml

```diff
@@ -1,12 +1,15 @@
 language: node_js
 node_js:
-  - "6"
```

Comment:
Tests fail in node 6, I think it's because of updating mocha but maybe we can just drop node 6 support here?

Reply:
i don't object